bots/run-queue

```diff
@@ -52,6 +53,7 @@ def main():
     connection = pika.BlockingConnection(pika.ConnectionParameters(
         host=host,
         port=int(port),
+        heartbeat_interval=60,
```

Comment:
Do you want to leave this in the final commit? It seems a bit more practical (and less redundant) to me to set it in rabbitmq.conf on the deployment.